requirements-ci.txt

```
@@ -1,4 +1,4 @@
 -r requirements-dev.txt
 -r requirements-prod.txt
 
-coverage==5.5
+coverage==6.3.2
```

```suggestion
coverage==6.3.2 # needs to be also updated in .github/workflows/ci.yml
```
Pour que les prochains qui mettront à jour n'oublient pas :)